auditing/auditing-interceptor.go

```diff
@@ -28,7 +28,14 @@ func UnaryServerInterceptor(a Auditing, logger *zap.SugaredLogger, shouldAudit f
 		if !shouldAudit(info.FullMethod) {
 			return handler(ctx, req)
 		}
-		requestID := uuid.New().String()
+		var requestID string
+		if str, ok := ctx.Value(rest.RequestIDKey).(string); ok {
+			requestID = str
+		}
+		if requestID == "" {
+			requestID = uuid.New().String()
```

Comment:
```suggestion
			requestID = uuid.NewString()
```

Also for all other occurences